### PR TITLE
Updated settings page popups using material design and updated the material-ui branch with development code

### LINKED ISSUE #371: Add material design in the existing popups
**Description:**
As of now, we are using react-bootstrap, replace the effect of bootstrap with material design

**Additional context**
Working on it.

### PR DESCRIPTION
**Pull request by Rupeshiya**
Updated Settings popups with material UI design enhancements
Fixes #371 
**Screenshots**
name
username
emailup
ident
deactivate

@jaskirat2000 @vaibhavdaren Please review

### CODE REVIEW

**Comment by Rupeshiya** [comment]
@vaibhavdaren @jaskirat2000 Please review

**Comment by jaskirat2000** [comment]
Here. Instead of "close" there should be "cancel" button and Save button should be adjacent to cancel.
Update identity doesn't look nice

**Comment by Rupeshiya** [comment]
@jaskirat2000 What changes should I do in the verify identity part?
Should I remove the icon and place the text instead or anything else?

**Comment by jaskirat2000** [comment]
Can you make it animated like this? https://images.app.goo.gl/W6VZXaCDk18zgS2t5 along with some text in Bold like "Updated, Thanks!"

Also I just wanted to know that why purple color is coming? We have blue color in donut rather than purple.

**Comment by Rupeshiya** [comment]
@jaskirat2000 Updated with the changes you stated above 
Please check here 
8e8b7c5e-9a23-45b8-a3fc-855e3d1d8b43

File: src/user/pinned-posts/pinned-posts.scss [hunk 1]
```diff
@@ -3,6 +3,11 @@
   min-height: 100vh;
   height: auto;
   font-family: Muli, sans-serif;
+  h1{
+    font-size: 26px;
+    font-weight: bold;
+    font-family: Muli, sans-serif;
```
Comment (vaibhavdaren) [inline src/user/pinned-posts/pinned-posts.scss]: use inheritance in css

File: src/user/pinned-posts/pinned-posts.scss [hunk 2]
```diff
@@ -16,6 +21,12 @@
   }
   .promotions {
     flex: 1;
-    padding: 20px 12px 20px 0px;
+    margin: 0px 20px;
```
Comment (vaibhavdaren) [inline src/user/pinned-posts/pinned-posts.scss]: will resizing work

File: src/user/dashboard/settings/layouts/SettingSidebar.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+              <b><MdNotifications size={iconsSize}/>Notification</b>
+            </NavLink>
+          </ListGroup.Item>
+          <ListGroup.Item className={posts ? 'active' : 'inactive'}>
+            <NavLink to="allPosts" className="link">
+              <b><MdRssFeed size={iconsSize}/>Posts</b>
+            </NavLink>
+          </ListGroup.Item>
+          <ListGroup.Item className={blocked ? 'active' : 'inactive'}>
+            <NavLink to="/blocked" className="link">
```
Comment (vaibhavdaren) [inline src/user/dashboard/settings/layouts/SettingSidebar.js]: for loop

File: src/common/Popups.js [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+          <Modal.Title>Update { " " + option }</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          {Boolean(option === "name") ? updateName : null}
+          {Boolean(option === "email") ? updateEmail: null}
+          {Boolean(option === "username") ? updateUsername: null}
+          {Boolean(option === "account") ? deactivateAccount: null}
+          {Boolean(option === "identity") ? checkVerification : null}
+        </Modal.Body>
+        <Modal.Footer>
```
Comment (vaibhavdaren) [inline src/common/Popups.js]: use for loop store keys in a array

Comment (vaibhavdaren) [inline src/common/Popups.js]: use for loop store keys in an array

Comment (Rupeshiya) [reply src/common/Popups.js]: @vaibhavdaren I have not done these changes, these changes are there as I have just fetched the development branch code and started working on that code.
So should I resolve this here or in development branch.

Comment (Rupeshiya) [reply src/common/Popups.js]: The problem was that the Material-UI branch does not have an updated code of development branch

Comment (Rupeshiya) [reply src/common/Popups.js]: @vaibhavdaren

Comment (vaibhavdaren) [reply src/common/Popups.js]: Ok.

Comment (vaibhavdaren) [reply src/common/Popups.js]: Start of here slowly we will move to development branch

Comment (Rupeshiya) [reply src/common/Popups.js]: cool, I will do the required changes here!

File: src/user/dashboard/settings/layouts/SettingSidebar.js (hunk 3 above)
Comment (Rupeshiya) [reply src/user/dashboard/settings/layouts/SettingSidebar.js]: done!

File: src/common/Popups.js (hunk 4 above)
Comment (Rupeshiya) [reply src/common/Popups.js]: done!

File: src/user/pinned-posts/pinned-posts.scss (hunk 2 above)
Comment (Rupeshiya) [reply src/user/pinned-posts/pinned-posts.scss]: fixed!

File: src/user/pinned-posts/pinned-posts.scss (hunk 1 above)
Comment (Rupeshiya) [reply src/user/pinned-posts/pinned-posts.scss]: fixed!

**Comment by Rupeshiya** [comment]
@vaibhavdaren  I have updated it, please let me know if still anything needs to be done!

**Pull request merged**